### DISCUSSION TITLE
Mismatching versions?

### DISCUSSION THREAD

**Discussion by nuzzles** [wasm-bindgen/wasm-bindgen#3532] [Q&A] [answered]
I keep getting this error:

```
 Running `wasm-server-runner /Users/simbleau/git/matchbox/target/wasm32-unknown-unknown/debug/simple_example.wasm`
Error: 

it looks like the Rust project used to create this wasm file was linked against
version of wasm-bindgen that uses a different bindgen format than this binary:

  rust wasm file schema version: 0.2.87
     this binary schema version: 0.2.84

Currently the bindgen format is unstable enough that these two schema versions
must exactly match. You can accomplish this by either updating the wasm-bindgen
dependency or this binary.

You should be able to update the wasm-bindgen dependency with:

    cargo update -p wasm-bindgen

or you can update the binary with

    cargo install -f wasm-bindgen-cli

if this warning fails to go away though and you're not sure what to do feel free
to open an issue at https://github.com/rustwasm/wasm-bindgen/issues!
```

Yet, I've done `cargo install wasm-bindgen@0.2.87 --force` and the steps listed to no avail.

**Comment by Liamolucko** [comment, marked as answer]
`wasm-server-runner` is the culprit here - it seems to be embedding a fixed version of `wasm-bindgen-cli-support` inside itself, and so updating your system version of the `wasm-bindgen` CLI has no effect.

Re-running `cargo install wasm-server-runner` should fix this for the moment, since it's not pinned to a particular version of `wasm-bindgen-cli-support` and so `cargo install` will pick the latest version of it to use. You'll have to repeat that every time you update the `wasm-bindgen` library though, and I don't think there's any good way to downgrade it to an older version of `wasm-bindgen-cli-support` if you need to.

**Comment by jaques-sam-tlv** [comment]
Not working here, it seems the runner doesn't influence the process at all. My difference is:
```
  rust Wasm file schema version: 0.2.106
     this binary schema version: 0.2.99 (04ca6f34a)
```
